javav2/example_code/cloudwatch/Readme.md

```diff
@@ -1,90 +1,98 @@
-Before running the Amazon CloudWatch JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. For example, you define an alarm name for various tests. If you do not define all values, the JUnit tests fail.
+### Prerequisites
 
-Define these values to successfully run the JUnit tests:
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html). 
 
-- **logGroup** - The name of the log group to use. For example, **testgroup**.
+To run these examples, you must have the following three JSON files: jsonWidgets.json, CloudDashboard.json, and settings.json. Find these files in this GitHub repository. The CloudWatch scenario depends on these files.
+ 
+ **Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
 
-- **alarmName** – The name of the alarm to use. For example, **AlarmFeb**.
+ ## Test the examples
+ 
+ ⚠️ Running the tests might result in charges to your AWS account.
 
-- **instanceId** – The ID of the instance to use. You can obtain this value from the AWS Management Console. For example, **ami-04300000000**.
+You can test the Java code example for Amazon CloudWatch by running a test file named **CloudWatchTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
 
-- **streamName** - The name of the stream to use. This value is used to retrieve log events.
+To successfully run the tests, define the following values in the test:
 
-- **ruleResource** – The Amazon Resource Name (ARN) of the user who owns the rule. You can obtain this value from the AWS Management Console.  
+- **logGroup** - The name of the log group to use. For example, **testgroup**.
+- **alarmName** – The name of the alarm to use. For example, **AlarmFeb**.
+- **instanceId** – The ID of the instance to use. Get this value from the AWS Management Console. For example, **ami-04300000000**.
+- **streamName** - The name of the stream to use. This value is used to retrieve log events.
+- **ruleResource** – The Amazon Resource Name (ARN) of the user who owns the rule. Get this value from the AWS Management Console.  
 -  **filterName**  - The name of the filter to use.
-
 - **destinationArn** - The ARN of the destination. This value is used to create subscription filters.
-
 - **roleArn** - The ARN of the user. This value is used to create subscription filters.
-
 - **filterPattern** - The filter pattern. For example, **Error**.
+- **myDateSc** - The start date to use to get metric statistics in the scenario test. (For example, 2023-01-11T18:35:24.00Z). 
```

Comment:
Not all of these look like they belong to this example. Did FilterPattern, DestinationArn, etc (the older ones) get used in this example, or are they from CloudWatch Logs examples?

Reply:
They are for single service actions - not scenario